CHANGES.md

```diff
@@ -3,6 +3,8 @@ merlin NEXT_VERSION
 
   + editor modes
     - vim: load merlin when Vim is compiled with +python3/dyn (e.g. MacVim)
+  + merlin binary
+    - Fix a follow-up issue to the preference of non-ghost nodes introduced in #1660 (#1690, fixes #1689)
```

Comment:
We usually put the merlin binary section first.

Reply:
Fixed